cmd/commands/git-source.go

```diff
@@ -111,6 +111,7 @@ func NewGitSourceCreateCommand() *cobra.Command {
 	var (
 		insCloneOpts *git.CloneOptions
 		gsCloneOpts  *git.CloneOptions
+		initRepo   bool
```

Comment:
this should be indented. It does it automatically for me when I press command-S. It doesn't do it for you?